rustls/src/client/client_conn.rs

```diff
@@ -603,6 +603,11 @@ pub trait ClientQuicExt {
         let ext = match quic_version {
             quic::Version::V1Draft => ClientExtension::TransportParametersDraft(params),
             quic::Version::V1 => ClientExtension::TransportParameters(params),
+            _ => {
+                return Err(Error::General(
+                    "QUIC unsupported version".into(),
+                ));
+            }
```

Comment:
Should draft 23 not also use `TransportParametersDraft`?

Reply:
I do not know : I just introduced the minimum set of changes to get my use case to work.
In facts, I would rather not need to change rustls and be able to use `rustls::quic::Keys::initial` with my own defined `Version`/original_salt

What is the best way to do so ?